app/mailers/observer_mailer.rb

```diff
@@ -1,28 +1,43 @@
 class ObserverMailer < ApplicationMailer
-  def on_observer_added(observation, reason)
+  def observer_added_confirmation(observation, reason)
```

Comment:
since the `observer_removed_*` action can be performed by either the observer themselves or some other user, should we leave that as `confirmation` or change?